airflow/providers/google/cloud/operators/bigquery.py

```diff
@@ -1195,36 +1200,24 @@ def execute(self, context) -> None:
         else:
             schema_fields = self.schema_fields
 
-        if schema_fields and self.table_resource:
-            self.table_resource["externalDataConfiguration"]["schema"] = schema_fields
```

Comment:
Does removing this logic break backward compatibility?

Reply:
Considering it was incorrectly introduced and table creation via table resource is not properly supported, I don't think this will break backwards compatibility

Reply:
Agree.